moonstreamapi/moonstreamapi/routes/subscriptions.py

```diff
@@ -624,7 +638,7 @@ async def update_subscriptions_handler(
         f"{key}:{value}"
         for tag in subscription_required_fields
         for key, value in tag.items()
-        if key not in MOONSTREAM_ENTITIES_RESERVED_TAGS
+        if key not in MOONSTREAM_ENTITIES_RESERVED_TAGS and key != "instance_name"
```

Comment:
Why do we need here check instance name, since `instance_name` already in reserved tags?